src/main/webapp/WEB-INF/jsp/scripts.jsp

```diff
@@ -18,10 +18,19 @@
     under the License.
 
 --%>
+
```

Comment:
please be aware that tooltip() is utilised within bootstrap, and jquery, therefore a way of ensuring the javascript does not error out will need to be created to ensure the client console does not get flooded with tooltip is not defined references. This commented out code, if it were to be commented back in,  should fix the tooltip() issue, but i can not confirm it, and would occur at the common javascript level.